### PR TITLE
Added a new book

### PR DESCRIPTION
**Pull request by thewhitetulip**

### CODE REVIEW

**Comment by eshellman** [comment]
Great addition!

File: free-programming-books.md [hunk 1]
```diff
@@ -296,7 +296,7 @@
 * [Cellular Automata Books](http://uncomp.uwe.ac.uk/genaro/Cellular_Automata_Repository/Books.html)
 
 
-#### Cloud Computing
+#### Cloud Computinga
```
Comment (aplanas) [inline free-programming-books.md]: Computinga?

**Comment by thewhitetulip** [comment]
Thank you @eshellman!

**Comment by vhf** [comment]
Thanks @thewhitetulip !

**Pull request merged**